seqr/views/status.py

```diff
@@ -26,7 +27,7 @@ def status_view(request):
     try:
         redis.StrictRedis(host=REDIS_SERVICE_HOSTNAME, socket_connect_timeout=3).ping()
     except Exception as e:
-        dependent_services_ok = False
+        secondary_services_ok = False
```

Comment:
Is Redis also a secondary service? It looks like a dependent one.

Reply:
seqr runs fine if redis is down, just a bit slower